README.md

```diff
@@ -228,7 +221,7 @@ All rights reserved.
     "Travis Jobs"
 
 [appveyor badge]:
-    <https://ci.appveyor.com/api/projects/status/ikn2l4u1xsume63u/branch/master?svg=true>
+    <https://ci.appveyor.com/api/projects/status/xxxxxxTODOxxxxxx/branch/master?svg=true>
```

Comment:
Just post the link here, I will insert it.

Reply:
All I need is the random part of the URL, which in my case is `ikn2l4u1xsume63u`.

Reply:
Thank you. Added in commit 4d75b684092527d50d2a82e40e2c0bf4f34ae0be.